bootstrap/handlers/httpserver.go

```diff
@@ -116,8 +116,9 @@ func (b *HttpServer) BootstrapHandler(
 	}).HandlerFunc(HandlePreflight(bootstrapConfig.Service.CORSConfiguration))
 
 	server := &http.Server{
-		Addr:    addr,
-		Handler: b.router,
+		Addr:              addr,
+		Handler:           b.router,
+		ReadHeaderTimeout: 5 * time.Second, // G112: A configured ReadHeaderTimeout in the http.Server averts a potential Slowloris Attack
```

Comment:
Why is this part of this PR. Are you using newer version of the linter? 
@bnevis-i , are we ok with hard coded 5 secs for this?

Reply:
@lenny-intel Yes.  The number of seconds is somewhat arbitrary.  The important thing is to have a limit.